### PR TITLE
Update toc/book and tutorial

### PR DESCRIPTION
**Pull request by lamberta**
stage: cl/328835209

### CODE REVIEW

**Comment by lamberta** [comment]
> Would it make sense to add the [tutorial colab](https://github.com/quantumlib/qsim/blob/master/docs/qsimcirq_tutorial.ipynb) to the "Guide & Tutorials" section?

Done. Thanks

I moved and formatted the notebook a bit (add buttons, run through [nbfmt](https://github.com/tensorflow/docs/tree/master/tools/tensorflow_docs/tools), etc). But this will give you the gist of things as you add more tutorials.

Removed the notebook outputs so we can execute and test the notebook when we import for publishing (notebooks with outputs saved are published as-is and not tested).

Updated preview which doesn't have outputs: go/q-stage/qsim/tutorials/qsimcirq
But outputs will be added here when the regular import job is run (nightly, if changes).

File: docs/tutorials/qsimcirq.ipynb [hunk 1]
```diff
@@ -1,87 +1,139 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "5jqw8k0zBsb-"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.example.org/qsim/tutorials/qsimcirq\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on QuantumLib</a>\n",
```
Comment (95-martin-orion) [inline docs/tutorials/qsimcirq.ipynb:56]: Is this "example.org" link a placeholder until the doc site is public?

File: docs/tutorials/qsimcirq.ipynb [hunk 2]
```diff
@@ -1,87 +1,139 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/qsim/blob/master/docs/tutorials/qsimcirq.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/quantumlib/qsim/blob/master/docs/tutorials/qsimcirq.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/qsim/docs/tutorials/qsimcirq.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
```
Comment (95-martin-orion) [inline docs/tutorials/qsimcirq.ipynb:65]: These links don't currently work, but I guess they will once this PR is submitted?

File: docs/tutorials/qsimcirq.ipynb (hunk 1 above)
Comment (lamberta) [reply docs/tutorials/qsimcirq.ipynb:56]: That's right. This url isn't available, yet, so we'll have to update this link later (same for the other projects)

File: docs/tutorials/qsimcirq.ipynb (hunk 2 above)
Comment (lamberta) [reply docs/tutorials/qsimcirq.ipynb:65]: Correct. Once this is merged the Colab and GitHub links will work.
The Download button (`storage.googleapis.com`) will work after the first site import (non-preview).

**Comment by lamberta** [comment]
> there are a couple of links in the docsite that point to this page - we should redirect those to the docsite version now.

I couldn't find any links within this repo or quantumlib/Cirq, do you know where else?

**Comment by 95-martin-orion** [comment]
> > there are a couple of links in the docsite that point to this page - we should redirect those to the docsite version now.
> 
> I couldn't find any links within this repo or quantumlib/Cirq, do you know where else?

This was specifically referring to the docsite. I don't think any additional changes are required in the qsim or Cirq repos.

**Comment by lamberta** [comment]
Friendly ping to merge this, please cc: @balopat

**Pull request merged**